tripal/tripal.install

```diff
@@ -1010,21 +1010,21 @@ function tripal_uninstall_cloner() {
 }
 
 /**
- * Adds the tripal_pub_import table TEMPORARY
+ * Adds the tripal_pub_library_query table TEMPORARY
  */
 function tripal_update_8403() {
```

Comment:
Is this the correct number for the update hook?  Shouldn't it be something like 10401?

Reply:
Thanks, I missed this!